src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -96,43 +97,37 @@ private Response GetCalendarFeed()
 
 
 
-            foreach (var episode in episodes.OrderBy(v => v.AirDateUtc.Value))
+            foreach (var album in albums.OrderBy(v => v.ReleaseDate))
             {
-                if (premiersOnly && (episode.SeasonNumber == 0 || episode.EpisodeNumber != 1))
-                {
-                    continue;
-                }
+                //if (premiersOnly && (album.SeasonNumber == 0 || album.EpisodeNumber != 1))
+                //{
+                //    continue;
+                //}
 
-                if (tags.Any() && tags.None(episode.Series.Tags.Contains))
+                if (tags.Any() && tags.None(album.Artist.Tags.Contains))
                 {
                     continue;
                 }
 
                 var occurrence = calendar.Create<Event>();
-                occurrence.Uid = "NzbDrone_album_" + episode.Id;
-                occurrence.Status = episode.HasFile ? EventStatus.Confirmed : EventStatus.Tentative;
-                occurrence.Description = episode.Overview;
-                occurrence.Categories = new List<string>() { episode.Series.Network };
+                occurrence.Uid = "NzbDrone_album_" + album.Id;
+                //occurrence.Status = album.HasFile ? EventStatus.Confirmed : EventStatus.Tentative;
+                //occurrence.Description = album.Overview;
+                //occurrence.Categories = new List<string>() { album.Artist. };
 
                 if (asAllDay)
```

Comment:
Yes, the issue is album doesn't have a HasFile property. Since files are associated with Tracks. We need to come up property in album that tells us if we have all tracks or no.